### ISSUE TITLE
Missing `cd Build` to build instructions

### ISSUE THREAD

**Issue by nothingismagick** [leaf3d/leaf3d#3]
![screen shot 2018-04-30 at 17 22 40](https://user-images.githubusercontent.com/35242872/39435256-2cdb2aa6-4c9b-11e8-81a7-c401c698dd37.png)

You have an oversight in your howto. It should read:

```bash
# Clone repo
$ git clone git@github.com:leaf3d/leaf3d.git

# Build project
$ cd leaf3d
$ mkdir Build
$ cd Build
$ cmake ..
$ cmake --build .
```

**Comment by zuck** [comment]
Hi @nothingismagick, nice catch, thank you!

Could you please provide feedback about necessary steps to build **leaf3d** on Mac platform (i.e. how to install dependencies)?

Thanks.

**Comment by nothingismagick** [comment]
Well, I may have spoken too soon. It built, but the binaries in Release/Leaf.app just show blank windows...

And really, I just followed exactly the same instructions as you listed.

**Comment by zuck** [comment]
Issue #3 fixed by 390dbe51f578b22486ae5ca6726461c08229ba8c. Thanks, @nothingismagick.

Please open a new issue to track build issues on Mac.
